lib/mrsk/cli/main.rb

```diff
@@ -37,6 +37,9 @@ def deploy
         say "Ensure app can pass healthcheck...", :magenta
         invoke "mrsk:cli:healthcheck:perform", [], invoke_options
 
+        say "Stop stale containers...", :magenta
+        invoke "mrsk:cli:app:stop_stale_containers", [], invoke_options
+
```

Comment:
Perhaps `boot` should stop all old containers rather than just the previous version?

Reply:
Yes!